subsys/net/l2/ieee802154/ieee802154_mgmt.c

```diff
@@ -865,7 +876,8 @@ static int ieee802154_get_parameters(uint64_t mgmt_request,
 		return -EINVAL;
 	}
 
-	if (mgmt_request == NET_REQUEST_IEEE802154_GET_EXT_ADDR) {
+	if (mgmt_request == NET_REQUEST_IEEE802154_GET_EXT_ADDR ||
+		mgmt_request == NET_REQUEST_IEEE802154_GET_COORD_EXT_ADDR) {
```

Comment:
Please fix the alignment
```suggestion
	if (mgmt_request == NET_REQUEST_IEEE802154_GET_EXT_ADDR ||
	    mgmt_request == NET_REQUEST_IEEE802154_GET_COORD_EXT_ADDR) {
```